crates/conductor_lib/src/conductor/base.rs

```diff
@@ -312,9 +302,7 @@ impl Conductor {
     }
 
     pub fn p2p_bindings(&self) -> Option<Vec<String>> {
-        self.network_spawn
-            .as_ref()
-            .map(|spawn| spawn.p2p_bindings.clone())
+        None
```

Comment:
I think we need a comment here to explain why this is `None` and how something might be addable back in here in the future?

Reply:
actually, turns out this function isn't used anywhere - removed